src/matchers/pass/index.js

```diff
@@ -0,0 +1,10 @@
+import predicate from './predicate';
+
+const passMessage = message => {
+  if (message) return () => message;
+  else return () => 'Passed!';
```

Comment:
Should we have this be something a little bit more descriptive, e.g. 'passed by .pass() assertion.'

Reply:
Happy to change it to whatever you'd like!